src/proxyvisor.coffee

```diff
@@ -0,0 +1,316 @@
+Promise = require 'bluebird'
+dockerUtils = require './docker-utils'
+{ docker } = dockerUtils
+express = require 'express'
+fs = Promise.promisifyAll require 'fs'
+{ resinApi } = require './request'
+knex = require './db'
+_ = require 'lodash'
+deviceRegister = require 'resin-register-device'
+randomHexString = require './lib/random-hex-string'
+utils = require './utils'
+device = require './device'
+bodyParser = require 'body-parser'
+request = Promise.promisifyAll require 'request'
+appConfig = require './config'
```

Comment:
Renamed this here because otherwise assigning config below would shadow it.
CoffeeScript...